packages/react/src/components/Masthead/Masthead.js

```diff
@@ -145,11 +145,9 @@ const Masthead = ({
        * L0 will hide/show only in the top 25% of the viewport.
        *
        */
-      if (mastheadL1Ref.current != null) {
-        setIsMastheadSticky(
-          root.pageYOffset > root.innerHeight * hideTopnavThreshold
-        );
-      }
+      setIsMastheadSticky(
+        root.pageYOffset > root.innerHeight * hideTopnavThreshold
```

Comment:
I wonder if the calculation needs to be tweaked here, this statement becomes true too soon after user starts scrolling

Reply:
I think the calculation is fine, because it is getting 25% of the page and comparing with current page position. if it is too soon, I think that need to change the threshold (25%).